content/python/concepts/built-in-functions/terms/memoryview/memoryview.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: 'memoryview()'
+Description: 'Creates a 'memory view' object that allows Python code to access the internal data of an object's buffer without making a copy of it'
+Subjects:
+  - 'Data Types'
+  - 'Computer Science'
+  - 'Built-in Functions'
+Tags:
+
+CatalogContent:
+---
+
+Creates a `memory view` object that allows Python code to access the internal data of an object's buffer without making a copy of it.
+
+## Syntax
+
+```python
+obj = bytes([65, 66, 67, 68, 69])
+memoryview(obj)
+
+# output <memory at 0x1112b7e80>
+```
+## Example 1
+
+`memory view` can only be used on objects that support the buffer protocol and have a readable or writable buffer interface, such as `byte`, `bytearray`, `array.array`, `numpy` arrays, and other objects created using the struct module. `list` and `str` objects don't support buffer protocol.
+
+```python
+l = [1, 2, 3, 4, 5, 6]
+memoryview(l)
+#output TypeError: memoryview: a bytes-like object is required, not 'list'
```

Comment:
Instead of putting the output in comments in the code, we'd prefer it to be in a separate ` ```shell` block after the example code. Like this:

````
This code results in the following output:

```shell
Output from the code.
```
````